solr_manager/solr_files/solr_init.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
This one could be removed; isn't being used.

Reply:
removed it